### DISCUSSION TITLE
Should I use `matchDepNames` or `matchPackageNames`?

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#25233] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

I get these message in my Renovate logs:

> INFO: Use `matchDepNames` instead of `matchPackageNames`

I suppose the end result I'd like is for this to go away 😄 

### What you tried so far.

I considered replacing each use of `matchPackageNames` with `matchDepNames` but I'm not sure if that's what the message quoted above means. The log detail quotes this config snippet:

```json
{
  "packageRule": {
    "groupName": "PyYAML and its types",
    "matchDatasources": [
      "pypi"
    ],
    "matchPackageNames": [
      "PyYAML",
      "types-PyYAML"
    ]
  }
  "packageName": "pyyaml"
  "depName": "PyYAML"
}
```

I'm not clear if that's the only place I should make the proposed change, or if it's just an example.

### Relevant debug logs

<details><summary>Logs</summary>

```
INFO: Use matchDepNames instead of matchPackageNames
{
  "packageRule": {
    "groupName": "PyYAML and its types",
    "matchDatasources": [
      "pypi"
    ],
    "matchPackageNames": [
      "PyYAML",
      "types-PyYAML"
    ]
  }
  "packageName": "pyyaml"
  "depName": "PyYAML"
}
```

</details>

**Comment by rarkins** [comment, marked as answer]
If you're matching against `depName`, use `matchDepNames`, else if you're matching against `packageName` then use `matchPackageNames`. Currently `matchPackageNames` matches against both `packageName` and `depName` but it's warning if the latter is used and will deprecate it.

You can look at `packageFiles with updates` to see `depName`/`packageName` values.

**Reply by allanlewis** [reply]
Thanks, @rarkins - if that's the case then I find the message a little confusing, but given it's not actually a warning I'll just ignore it.

**Reply by rarkins** [reply]
We will change this from INFO to WARN and then remove the functionality, so I advise you don't ignore it. The message you see should include metadata with it to be more understandable. Are you seeing that?

```
      logger.once.info(
        { packageRule, packageName, depName },
        'Use matchDepNames instead of matchPackageNames'
      );
```
